main/main.cpp

```diff
@@ -954,8 +954,13 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 #endif
 
 	GLOBAL_DEF("logging/file_logging/enable_file_logging", false);
-	GLOBAL_DEF("logging/file_logging/log_path", "user://logs/log.txt");
-	GLOBAL_DEF("logging/file_logging/max_log_files", 10);
+	// Only file logging by default on desktop platforms as logs can't be
+	// accessed easily on mobile/Web platforms (if at all).
+	// This also prevents logs from being created for the editor instance, as feature tags
+	// are disabled while in the editor (even if they should logically apply).
+	GLOBAL_DEF("logging/file_logging/enable_file_logging.pc", true);
+	GLOBAL_DEF("logging/file_logging/log_path", "user://logs/log.log");
```

Comment:
`logs/log.log` starts being a bit redundant :) Maybe `godot.log`?

Reply:
Maybe, but I don't know if calling it `godot.log` is a good idea as that name makes less sense in an exported project.

Reply:
Unity uses `Player.log`, but I guess they call their runtime "Unity Player". The same wouldn't make sense for us.

I guess it could be `game.log`, but IMO `godot.log` is OK, the base contents will be Godot-related output like Godot version and GPU, and then any errors that the engine raises. Users can rename it if they prefer to give it a game-related name.